### PR TITLE
bump: pytorch-lightning >=2.0 & enable Torch 2.0+ (contd.)

### PR DESCRIPTION
**Pull request by manujosephv**
A PR to accomodate a few more changes and add on to @Borda PR #297

----
:books: Documentation preview :books:: https://pytorch-tabular--318.org.readthedocs.build/en/318/

### CODE REVIEW

**Comment by manujosephv** [comment]
@Borda For some reason, the testing suite isnt running... can you help?

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -6,50 +6,50 @@ jobs:
...
-                  path: tests/.datasets
-                  key: test-datasets
+    strategy:
+        fail-fast: false
+        matrix:
+            os: [ubuntu-latest]
+            python-version: ["3.8", "3.9", "3.10"]
+            include:
+                - { os: "ubuntu-20.04", python-version: "3.8", requires: "oldest" }
+                - { os: "ubuntu-20.04", python-version: "3.9", requires: "oldest" }
```
Comment (Borda) [inline .github/workflows/testing.yml:16]:
```suggestion
        strategy:
            fail-fast: false
            matrix:
                os: [ubuntu-latest]
                python-version: ["3.8", "3.9", "3.10"]
                include:
                    - { os: "ubuntu-20.04", python-version: "3.8", requires: "oldest" }
                    - { os: "ubuntu-20.04", python-version: "3.9", requires: "oldest" }
```
seems like syntax issue as per error message:
```
No steps defined in `steps` and no workflow called in `uses` for the following jobs: pytest, strategy
```

**Comment by Borda** [comment]
> @Borda For some reason, the testing suite isnt running... can you help?

you have a merge collision :)
```
No steps defined in `steps` and no workflow called in `uses` for the following jobs: pytest, strategy
```

**Comment by manujosephv** [comment]
Complete noob in CI/CD.. 😆 But, yes, I figured out it was an indentation issue. I might have messed it up when I merged some conflicts.. It's running now..

**Pull request merged**